### ISSUE TITLE
Designer-Element Properties]: keyboard focus is not in sync with Narrator Announcement while navigating through the list items present under the 'Element Properties' Pane.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7639]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS Version: 21H2 (OS build 22000.318)
Browser Version: 105.0.1300.0 (Official build) dev(64-bit)
Screen Reader: Narrator
URL:  [Adaptive Cards](https://adaptivecards.io/)

Repro steps:
1.	Launch the URL: [Adaptive Cards](https://adaptivecards.io/)
2.	On the Narrator by using ‘control+ Windows+ Enter’ short key.
3.	Navigate to header control ‘Designer’ using tab key and activate it with enter key.
4.	Navigate to the 'Card Structure' through tab key and activate it and select ‘TextBlock’ element from the list.
5.	Navigate to the ‘Element Properties’ through tab key and activate it. 
6.	Navigate to the 'Spacing' combo box present under it and activate it using space key.
7.	Try to navigate through the list items present under it and observe the issue.

Actual Result:
keyboard focus is not in sync with Narrator when Narrator announcing Large 5/7.

Expected Result:
Keyboard focus should be in sync with Narrator and it should move to the ‘Large’ list item.

Notes:
Same issue replicates with all the ‘combo boxes’ present under ‘Element Properties’.

Observation:
Issue occurs with narrator only.

User Experience:
Screen reader users who are with low vision will get impacted if keyboard focus is not in sync with Narrator after which user will not be able to track or navigate the content properly.

MAS Reference Link:
(https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.02)

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0).

### Screenshots

https://user-images.githubusercontent.com/104482612/177137270-3cc8498a-d5ea-49b7-bc86-f1982b9c5fce.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
This issue is not repro on the environment, hence closing this issue.

Test Environment:
URL: https://adaptivecards.io/
OS Version: 23H2(OS Build 25931.1000)
Browser Version: Edge Dev (118.0.2060.1)
Screen Reader: Narrator
